src/hotspot/share/gc/parallel/psParallelCompact.cpp

```diff
@@ -2023,7 +2023,7 @@ static void mark_from_roots_work(ParallelRootType::Value root_type, uint worker_
   cm->follow_marking_stacks();
 }
 
-static void steal_marking_work(TaskTerminator& terminator, uint worker_id) {
+void steal_marking_work(TaskTerminator& terminator, uint worker_id) {
```

Comment:
If this needs to be made non-file-scoped then it should be moved into some relevant class.

Reply:
Sorry, never mind, the minimal diff confused me about the context.  We're in class scope here, not at namespace scope.

Reply:
Never mind, I was confused by the minimal diff.  This is already at class scope, not at file scope.